.github/workflows/pre-commit.yaml

```diff
@@ -29,7 +29,11 @@ jobs:
         run: |
           helm plugin install https://github.com/jtyr/kubeconform-helm --version v0.1.12
 
+      - name: Checkout branch/fork
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
       - name: Run pre-commit
         uses: Alfresco/alfresco-build-tools/.github/actions/pre-commit@a288b9efdaa50413573b2e130896d906478b8e50 # v5.31.0
         with:
           auto-commit: "true"
+          skip_checkout: "true"
```

Comment:
the checkout handling is needed for the auto-commit, I guess it's simpler to disable auto-commit when pr come from a forked repo